### DISCUSSION TITLE
500 (Internal Server Error) when open the page

### DISCUSSION THREAD

**Discussion by naga-wb** [vercel/vercel#5200] [Help]
Hi, I moved part of my project code to nextjs(v9.5.3). I am receiving 500 internal server error when I opened the page in browser. I didn't see any logs in console or terminal. 

I don't create any config file. Should I have one to resolve this issue?

OS: Ubuntu 20.04.1
OS Type: 64 bit

**Comment by paulogdm** [comment]
A 500 error is always caused by application errors or code-related issues. Please refer to https://vercel.com/knowledge/where-are-vercel-logs
